src/frontend/src/views/Data/cellRenderers.tsx

```diff
@@ -11,12 +11,14 @@ import {
   RowContent,
   TreeRowContent,
 } from "src/common/components/library/data_table/style";
+import { NewTabLink } from "src/common/components/library/NewTabLink";
 import { createTableCellRenderer } from "src/common/utils";
 import { datetimeWithTzToLocalDate } from "src/common/utils/timeUtils";
 import { isUserFlagOn } from "src/components/Split";
 import { USER_FEATURE_FLAGS } from "src/components/Split/types";
 import { CZ_BIOHUB_GROUP } from "src/views/Data/constants";
 import { LineageTooltip } from "./components/SamplesView/components/SamplesTable/components/LineageTooltip";
+import { StatusChip } from "./components/StatusChip";
```

Comment:
From what I can tell, this is a new component. Did you accidentally leave it out of the commit?

Reply:
yes!! thank you i always forget to add the new files

Reply:
added